src/bindings/python/requirements_test.txt

```diff
@@ -40,3 +40,6 @@ tox
 types-pkg_resources
 wheel
 singledispatchmethod
+torch
+torchvision
+pillow
```

Comment:
I suppose it's better to test via tox.ini files, where we can run different categories of tests in different environments. 
Currently, we are adding more and more to our requirements.txt files and it complicates future support because of conflicting dependencies. 

I'm not suggesting to change the approach in this specific PR, but rather a general consideration - drop requirements.txt for tests and rely only on tox.ini files.

Reply:
I think it's a good idea, let's continue it in 115462.